provision_app_server.sh

```diff
@@ -1,26 +1,41 @@
 source /tmp/load_variables.sh
 
 sudo apt-get update -y
-sudo apt-get install build-essential -y 
+sudo apt-get install build-essential -y
 sudo apt-get install git -y
 sudo apt-get install rubygems -y
 sudo apt-get install rubygems-integration -y
 
-sudo iptables -t nat -A PREROUTING -p tcp --dport 80 -j REDIRECT --to-port 8080
+# SSL
+wget https://dl.eff.org/certbot-auto
+chmod a+x certbot-auto
 
-sudo useradd -m hellogov
+# FIXME: don't hard code domain name
```

Comment:
Tiny thing, but you can remove this comment if you want. I forgot to when I changed to using $DOMAIN_NAME.